mods/cnc/rules/defaults.yaml

```diff
@@ -122,9 +122,12 @@
 		CloakDelay: 90
 		CloakSound: trans1.aud
 		UncloakSound: trans1.aud
-		RequiresCondition: cloak
+		RequiresCondition: cloak && !uncloak
 	ExternalConditions@CLOAK:
 		Conditions: cloak
+	GrantConditionOnDamageState@UNCLOAK:
+		Condition: uncloak
```

Comment:
If this were on the standard default template I would agree, however it is only included on and used by actors that can pick up cloak crates.  Giving it an even more general name would encourage people to break this encapsulation.